sdk/monitor/monitor-opentelemetry-exporter/src/export/statsbeat/customerStatsbeat.ts

```diff
@@ -0,0 +1,474 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import type { BatchObservableResult, Meter, ObservableGauge } from "@opentelemetry/api";
+import { diag } from "@opentelemetry/api";
+import type { PeriodicExportingMetricReaderOptions } from "@opentelemetry/sdk-metrics";
+import { MeterProvider, PeriodicExportingMetricReader } from "@opentelemetry/sdk-metrics";
+import type { AzureMonitorExporterOptions } from "../../index.js";
+import * as ai from "../../utils/constants/applicationinsights.js";
+import { StatsbeatMetrics } from "./statsbeatMetrics.js";
+import type { CustomerStatsbeatProperties, StatsbeatOptions } from "./types.js";
+import { CustomerStatsbeat, DropCode, RetryCode } from "./types.js";
+import { CustomStatsbeatCounter, STATSBEAT_LANGUAGE, TelemetryType } from "./types.js";
+import { getAttachType } from "../../utils/metricUtils.js";
+import { AzureMonitorStatsbeatExporter } from "./statsbeatExporter.js";
+
+/**
+ * Class that handles customer-facing statsbeat metrics
+ * These metrics are sent to the customer's breeze endpoint
+ *
+ * Implements a singleton pattern to ensure only one set of customer statsbeat metrics
```

Comment:
Do you want to have a singleton here? can the user create multiple exporters sending data to different iKeys? if that is supported this will send the stats for the first exporter created

Reply:
Spec doesn't specify either way. @lzchen what are your thoughts on customer statsbeat supporting multiple iKeys?